src/Aspire.Hosting.AWS/AWSSDKConfigResource.cs

```diff
@@ -0,0 +1,16 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Amazon;
+using Aspire.Hosting.ApplicationModel;
+
+namespace Aspire.Hosting.AWS;
+
+internal sealed class AWSSDKConfigResource(string name) : Resource(name), IAWSSDKConfigResource
```

Comment:
This doesn't feel like a resource type, it feels like external input that might be parameterized.

Reply:
It is now just `IAWSSDKConfig` implemented by `AWSSDKConfig`. This does help in getting rid of the `name` constructor which was going to confused with the `profile` name.